### DISCUSSION TITLE
isLoading and isValidating is always "false"

### DISCUSSION THREAD

**Discussion by kyrylolvov** [vercel/swr#2993] [Q&A]
I use swr in the following way, the request works, the data on first render is undefined, then gets the response, however for some reason `isLoading` and `isValidating` is always `false`.

Anyone knows whats the reason for this?

```
  const { data, isLoading, isValidating, mutate } = useSWR(`${method}-${baseUrl}-${route}`, null, {
    revalidateOnMount: false,
    shouldRetryOnError: false,
    revalidateOnFocus: false,
  });
```

**Comment by its-me-sv** [comment]
Still no replies? Anyone fixed it?
